docs/Deploying-Pega-on-AKS.md

```diff
@@ -509,6 +512,18 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
 
 3. Save the file.
 
+### Changes in Backend setting if tiers.service.tls.domain is `true`
+
+If the setting `tiers.service.tls.enabled` is set to `true` and the certificate in the keystore is issued by a valied CA authority, no additional steps is required.
+else if the certificate in the keystore is a self-signed or not issued by a valid certificate authority, the following additional configuration is required in Azure application gateway,
+
+- Go to the Application Gateway created
+- In the left side pane, under **Settings**, select **Backend Settings** option
+- Under **Trusted Root Certificate**, select **No** for **Use well known CA certificate** option
+- A new section appears, click **Create New**
+- Upload your CA certificate as .cer file in the **CER certificate** field and give a name for the certificate in **Cert name** field.
+- Click **Save** to update the backend settings
```

Comment:
A few small tweaks ;-)
- Go to the Application Gateway you created in AKS.
- In the left side pane, under **Settings**, select the **Backend Settings** option.
- Under **Trusted Root Certificate**, select **No** for **Use well known CA certificate** option.
- In the new CA certificate section, click **Create New**.
- Upload your CA certificate as .cer file in the **CER certificate** field and give a name for the certificate in **Cert name** field.
- Click **Save** to update the backend settings.